### ISSUE TITLE
[Bug]: config is not working in 5.0.0-alpha.1

### ISSUE THREAD

**Issue by vincent-scw** [sibiraj-s/ngx-editor#207]
#I'm submitting a

- [x] Regression (a behavior that used to work and stopped working in a new release)

## Description
```
<ngx-editor name="content" required [config]="editorConfig"
					[(ngModel)]="model.content"></ngx-editor>
```
editorConfig is not working in new version, but worked in previous version

### Version Information

```
ngx-editor: 5.0.0-alpha.1
angular: 9.1.2
```

### Browser

- [ ] Chrome - x.x.x

**Comment by sibiraj-s** [comment]
Rite now editor accepts only one cofiguration. thats toolbar. you can refer here, 
https://sibiraj.dev/ngx-editor/additional-documentation/configuration.html

can u post an example to reproduce your issue, a stackblitz or a repo may be.

**Comment by vincent-scw** [comment]
Please check this
[https://stackblitz.com/edit/angular-zxaejr](https://stackblitz.com/edit/angular-zxaejr)

**Comment by vincent-scw** [comment]
BTW, I am trying to upgrade ngx-editor from 3.3 to 5.0. I found the model binding has been changed from pure string to ProseMirror. Can you please write a little more to guide on this? Or these is a sample?

**Comment by sibiraj-s** [comment]
Fixed in #209 , will be published in alpha.2 shortly.
